docs/triggers/trigger-generic-inputs.md

```diff
@@ -0,0 +1,80 @@
+# Trigger Creation Workflow Definition Inputs
+
+When creating a trigger at runtime, a workflow definition can be passed into the `client.workflows.triggers.create` method which will allow for typeahead/autocomplete of the input parameters based on the definition's inputs. 
+
+## Defining A Workflow Recap
+
+Recall in the `manifest.ts` file of an app, a `workflow` can be defined using the `DefineWorkflow` method as follows:
+
+```ts
+export const TriggersWorkflow = DefineWorkflow({
+  callback_id: "triggers_workflow",
+  title: "Triggers Workflow",
+  description: "sample runthrough of a trigger creation",
+  input_parameters: {
+    properties: {
+      a_string: {
+        type: Schema.types.string,
+      },
+      b_string: {
+        type: Schema.types.string,
+      },
+      a_channel: {
+        type: Schema.slack.types.channel_id,
+      },
+    },
+    required: [],
+  },
+});
+```
+
+When a workflow is created in this manner, if contains a `workflow.definition` object. By passing this definition into the `create` method, the trigger will have access to information related to the workflow's definition.
```

Comment:
We shouldn't add documentation for how a SDK feature works in this repo as it will very likely become out of date. Link out to the SDK docs or to the public API docs instead.

The only exception is if we're actually using the values from this workflow in the examples below, which would mean that the examples should provide matching `input_parameters` based on the `required` field and that the value passed to `workflow` matches.